pokedex/src/components/PokemonInfo/AddPokemonInfoModal.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+const { useState, useEffect } = React;
+
+export default function AddModal({ show, handleClose, pokemonInfo, handleSave }) {
```

Comment:
nit: `Modal` or `Model?`

Reply:
Modal - it is a modal

Reply:
ah cool, not familiar with the terminology lol